Author: Daniel Kahn Gillmor <dkg@fifthhorseman.net>
Subject: bump dependencies to versions available in debian

upstream has already done this, with no necessary source changes:

See upstream commit de8583b632dc63b490b646624ffb7f956391d7a2

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -32,7 +32,7 @@
 default-features = false
 
 [dependencies.capnp-rpc]
-version = "0.13"
+version = "0.14"
 
 [dependencies.dirs]
 version = "4.0"
@@ -77,7 +77,7 @@
 features = ["rt-multi-thread", "io-util", "net"]
 
 [dependencies.tokio-util]
-version = "0.6"
+version = "0.7"
 features = ["compat"]
 [dev-dependencies.clap]
 version = "2.33"
